go.mod

```diff
@@ -6,119 +6,116 @@ require (
 	github.com/RHsyseng/operator-utils v0.0.0-20200213165520-1a022eb07a43
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
 	github.com/coreos/ignition v0.35.0
-	github.com/coreos/ignition/v2 v2.9.0
+	github.com/coreos/ignition/v2 v2.13.0
```

Comment:
why do we need this update? I'm asking because it's a bit unexpected to do it in the context of this PR. Perhaps we should have done it earlier?

Reply:
Those dependencies updated by go mod automatically after I bumped
`github.com/openshift/machine-config-operator`
from: `v0.0.1-0.20220203091316-d3010b34d344`
to: `v0.0.1-0.20230317184452-40575b862f7b`

Reply:
The bump itself is necessary because the `ContainerRuntimeConfig` object is relatively new under the  machine-config-operator api group

Reply:
Fair enough, we need the bump here so we're good.

Reply:
i'll add the rational for the bump in the commit message